permissions/plugin-frugal-testing.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "frugal-testing"
+github: "jenkinsci/frugal-testing-plugin"
+paths:
+- "org/jenkins-ci/plugins/frugal-testing"
```

Comment:
The path needs to follow the groupId/artifactId from your pom.xml (https://github.com/jenkinsci/frugal-testing-plugin/blob/master/pom.xml#L28).

So, you would need:

`- "io/jenkins/plugins/frugal-testing"`